src/main/java/lotto/domain/CountPrice.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+
+public enum CountPrice {
+    SIX_PRICE(6, new BigDecimal("2000000000")),
```

Comment:
```suggestion
    SIX_PRICE(6, new BigDecimal("2_000_000_000")),
```

이렇게 표기하면 가독성에 더 좋더라구요!